src/main/java/de/vill/model/expression/AvgAggregateFunctionExpression.java

```diff
@@ -21,6 +19,11 @@ public String toString(boolean withSubmodels, String currentAlias) {
 
     @Override
     public List<Expression> getExpressionSubParts() {
-        return Arrays.asList();
+        return Collections.emptyList();
+    }
+
+    @Override
+    public String getReturnType() {
+        return "number";
```

Comment:
This string is used very often. Maybe introduce a public String constant? Could also create a designated StringTable class if needed for more constants.